### PR TITLE
[INLONG-3201][TubeMQ] Improve the cluster query function

### PR DESCRIPTION
**Pull request by bluewang**
### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes https://github.com/apache/incubator-inlong/issues/3201

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-tubemq/tubemq-manager/pom.xml [hunk 1]
```diff
@@ -119,6 +119,11 @@
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
         </dependency>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.15</version>
+        </dependency>
```
Comment (EMsnap) [inline inlong-tubemq/tubemq-manager/pom.xml:126]: don't see the use of this import?

**Review by EMsnap — COMMENTED** [review]
PLAL

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java [hunk 2]
```diff
@@ -185,4 +190,95 @@ String queryInfo(
...
+        map.put("partitionCount", count);
+        return map;
+    }
+
+    /**
+     * query Consumer group count
+     * @param clusterId
+     * @return
+     */
+    public int gitConsumerGroupCount(long clusterId) {
```
Comment (EMsnap) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java:249]: getConsumer

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java [hunk 3]
```diff
@@ -185,4 +190,95 @@ String queryInfo(
...
+        String queryUrl = masterService.getQueryCountUrl(clusterId, TubeConst.QUERY_CONSUMER_GROUP_INFO);
+        int count = 0;
+        String groupData = masterService.queryMaster(queryUrl);
+        JsonObject jsonObject1 = gson.fromJson(groupData, JsonObject.class);
```
Comment (EMsnap) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java:253]: don't use number as variable name

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java [hunk 4]
```diff
@@ -132,9 +136,9 @@ public TubeMQResult queryCluster(@RequestParam(required = false) Integer cluster
         }
 
         MasterEntry masterNode = masterService.getMasterNode(clusterEntry.getClusterId());
-
+        Map<String, Integer> allCount = getAllCount(clusterId);
```
Comment (EMsnap) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java:139]: use map as return value is inappropriate

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java [hunk 5]
```diff
@@ -185,4 +190,95 @@ String queryInfo(
...
+    }
+
+    /**
+     * query consumer count
+     * @param clusterId
+     * @return
+     */
+    public int gitConsumerCount(long clusterId) {
+        String queryUrl = masterService.getQueryCountUrl(clusterId, TubeConst.QUERY_CONSUMER_INFO);
+        String s = masterService.queryMaster(queryUrl);
```
Comment (EMsnap) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java:271]: magic variable s

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java (hunk 4 above)
Comment (bluewang) [reply inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java:139]: Map is not used here, what is better to return？

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java [hunk 6]
```diff
@@ -185,4 +190,95 @@ String queryInfo(
...
+        int brokerSize = getBrokerSize(clusterId);
+        Map<String, Integer> mapCount = getTopicAndPartitionCount(clusterId);
+        int consumerGroupCount = getConsumerGroupCount(clusterId);
+        int consumerCount = getConsumerCount(clusterId);
+        Map<String, Integer> map = new HashMap<>();
+        map.put("brokerSize", brokerSize);
+        map.put("topicCount", mapCount.get("topicCount"));
+        map.put("partitionCount", mapCount.get("partitionCount"));
+        map.put("consumerGroupCount", consumerGroupCount);
+        map.put("consumerCount", consumerCount);
```
Comment (healchow) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java:203]: Maybe you can create a POJO class to set those fields?

**Pull request closed without merging**